mfaDisabled/function.js

```diff
@@ -1,39 +1,17 @@
 var GitHubApi = require('github');
-var message = require('lambda-cfn').message;
+var message = require('@mapbox/lambda-cfn').message;
 var d3 = require('d3-queue');
-var splitOnComma = require('lambda-cfn').splitOnComma;
-var getEnv = require('lambda-cfn').getEnv;
-
-module.exports.config = {
-  name: '2faDisabled',
-  runtime: 'nodejs4.3',
-  sourcePath: 'rules/2faDisabled.js',
-  parameters: {
-    'githubToken': {
-      'Type': 'String',
-      'Description': 'Github API token'
-    },
-    'githubOrganization': {
-      'Type': 'String',
-      'Description': 'Name of organization on Github'
-    },
-    'allowedList': {
-      'Type': 'String',
-      'Description': 'Comma separated list of Github accounts that do not require 2FA'
-    }
-  },
-  scheduledRule: 'rate(5 minutes)'
-};
+var splitOnComma = require('@mapbox/lambda-cfn').splitOnComma;
 
 module.exports.fn = function(event, context, callback) {
 
   var github = new GitHubApi({
     version: "3.0.0"
   });
 
-  var githubToken = getEnv('githubToken');
-  var githubOrganization = getEnv('githubOrganization');
-  var allowedList = splitOnComma(getEnv('allowedList'));
+  var githubToken = process.env.githubToken;
+  var githubOrganization = process.env.githubOrganization;
+  var allowedList = splitOnComma(process.env.allowedList);
```

Comment:
This reminds me - we may want to include some quick docs on how the `splitOnComma` function in lambda-cfn is useful here. This is another good candidate for centralized docs in /patrol since this function is even more useful for /patrol-rules-aws.